tomb-common/src/banyan_api/requests/core/auth/device_api_key/regwait/end.rs

```diff
@@ -48,72 +48,3 @@ impl ApiRequest for EndRegwait {
         true
     }
 }
-
-#[cfg(test)]
-#[cfg(feature = "fake")]
-pub mod test {
-    use super::*;
-    use crate::banyan_api::{
-        models::account::test::authenticated_client,
-        requests::core::auth::device_api_key::regwait::start::StartRegwait,
-        utils::generate_api_key,
-    };
-    use std::sync::Arc;
-    use tomb_crypt::prelude::PrivateKey;
-
-    #[tokio::test]
-    #[ignore]
-    async fn regwait_success() {
-        let mut client = authenticated_client().await;
-        let mut other_client = client.clone();
-        let (api_key, _pem) = generate_api_key().await;
-        let fingerprint_arc_bytes = api_key
-            .fingerprint()
-            .await
-            .expect("Failed to get fingerprint");
-        let fingerprint_bytes =
-            Arc::into_inner(fingerprint_arc_bytes).expect("Failed to get fingerprint bytes");
-        let fingerprint = fingerprint_bytes
-            .iter()
-            .fold(String::new(), |chain, byte| format!("{chain}{byte:02x}"));
-        let fingerprint_clone = fingerprint.clone();
-
-        let end_handle = tokio::spawn(async move {
-            std::thread::sleep(std::time::Duration::from_secs(1));
-            other_client
-                .call_no_content(EndRegwait {
-                    fingerprint: fingerprint_clone,
-                })
-                .await
-                .unwrap();
-        });
-        client
-            .call_no_content(StartRegwait { fingerprint })
-            .await
-            .unwrap();
-        end_handle.await.unwrap();
-    }
-
-    #[tokio::test]
-    #[ignore]
-    async fn regwait_fail() {
-        let mut client = authenticated_client().await;
-        let mut other_client = client.clone();
-        let end_handle = tokio::spawn(async move {
-            std::thread::sleep(std::time::Duration::from_secs(1));
-            other_client
-                .call_no_content(EndRegwait {
-                    fingerprint: "other_random_nonsense_string".to_string(),
-                })
-                .await
-                .expect_err("Expected an error");
-        });
-        client
-            .call_no_content(StartRegwait {
-                fingerprint: "random_nonsense_string".to_string(),
-            })
-            .await
-            .expect_err("Expected an error");
-        end_handle.await.unwrap();
-    }
-}
```

Comment:
this is an integration test Sam says we really shouldnt be testing this way, can revisit in the future

Reply:
agree, we'll put it back in the future